tests/test_to_cupy.py

```diff
@@ -22,49 +22,81 @@
 cp, has_cp = optional_import("cupy")
 
 
+@skipUnless(has_cp, "CuPy is required.")
 class TestToCupy(unittest.TestCase):
-    @skipUnless(has_cp, "CuPy is required.")
     def test_cupy_input(self):
-        test_data = cp.array([[1, 2], [3, 4]])
+        test_data = cp.array([[1, 2], [3, 4]], dtype=cp.float32)
```

Comment:
I am not familiar with Cupy APIs, may I know how to specify the target `device` for the array?

Thanks.

Reply:
CuPy arrays are  always on GPU! You can change the cuda device with `cp.cuda.Device(1).use()` or with a context manager!
Basically CuPy array on CPU is a NumPy array!

Reply:
`cp.cuda.Device(1).use()` is what I asked, I know Cupy array is on GPU, I just want to know how to specify GPU device in multi-GPUs case.

Thanks.

Reply:
I see. I like your forward thinking! ;)